Main.py

```diff
@@ -115,9 +115,11 @@ def main(args, seed=None, baked_server_options: Optional[Dict[str, object]] = No
         for item_name, count in world.start_inventory[player].value.items():
             for _ in range(count):
                 world.push_precollected(world.create_item(item_name, player))
-        for item_name, count in world.start_inventory_from_pool[player].value.items():
-            for _ in range(count):
-                world.push_precollected(world.create_item(item_name, player))
+
+        if player in world.start_inventory_from_pool:
```

Comment:
could we do `if player in getattr(world, "start_inventory_from_pool", {}):` and rip it out of the multiworld init?